File: Sopra/finished_project/sopra-fs19-group03-client/src/components/shared/models/outgoingMessages/CardCollectionMessage.js

import "../EnumsJS";



class CardCollectionMessage {
    constructor(id, collection){
        this.userId = id;
        this.cardCollection = collection;
    }
    getUserId(){return this.userId;}
    setUserId(id){this.userId = id;}
    getCollection(){return this.cardCollection;}
    setCollection(collection){this.cardCollection = collection;}
}
export default CardCollectionMessage;

File: Sopra/finished_project/sopra-fs19-group03-server/src/main/java/ch/uzh/ifi/seal/soprafs19/dto/incoming/CardCollectionMessage.java
package ch.uzh.ifi.seal.soprafs19.dto.incoming;

import ch.uzh.ifi.seal.soprafs19.constant.GameCard;

import java.util.ArrayList;

public class CardCollectionMessage implements Message {
    private Long userId;
    private ArrayList<GameCard> cardCollection;

    public Long getUserId() {
        return userId;
    }

    public ArrayList<GameCard> getCardCollection() {
        return cardCollection;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public void setCardCollection(ArrayList<GameCard> cardCollection) {
        this.cardCollection = cardCollection;
    }
}
